guide/english/sql/sql-inner-join-keyword/index.md

```diff
@@ -45,7 +45,7 @@ SELECT a.studentID, a.FullName, sat_score, a.programOfStudy, schoolEmailAdr
 FROM student AS a;
 ```
 
-student or LEFT table
+student or LEFT table-
```

Comment:
No reason to add the - at the end of studen or LEFT table.

However, if you want to fix something, add three backticks to the line after the following:
```
9 rows in set (0.00 sec)
```